components/automate-gateway/handler/iam/v2/introspect/introspect.go

```diff
@@ -232,13 +232,15 @@ func (a *AuthzServer) getAllowedMap(
 	// Fetches the id of the current user PLUS the team ids for that user
 	subjects := auth_context.FromContext(ctx).Subjects
 
-	resp, err := a.filterHandler.FilterAuthorizedPairs(ctx, subjects, mapByResourceAndAction, methodsInfo)
+	inputPairs := pairs.GetKeys(mapByResourceAndAction)
+
+	filteredPairs, err := a.introspectionHandler.FilterAuthorizedPairs(ctx, subjects, inputPairs)
```

Comment:
There's only one set of `mapByResourceAndAction` and `methodsInfo`, so they're no longer passed in, but retrieved in the introspection code.